Islandora/documentation
# Installation

**GitHub Issue**: https://github.com/Islandora/documentation/issues/1040

* Part of the current documentation sprint: https://docs.google.com/spreadsheets/d/1E-kRw9xE60CKK0qL1-phzeVKjEZu3qBKZ9d3LH1hDEE/edit#gid=0

# What does this Pull Request do?

Adds an installation component overview, adds complete documentation for a base manual installation, and revises/updates some of the documentation for the Ansible playbook installation.

# What's new?
* Component overview section
* Manual installation guide

# How should this be tested?
[Build and deploy as usual](https://islandora.github.io/documentation/technical-documentation/docs-build/) using `mkdocs` to follow along locally.

For the Ansible playbook section (now titled "Automatic Provisioning"), the instructions should still get you a functioning site from the Ansible playbook.

For the manual installation section:

* Spin up an Ubuntu 18.04 box using VirtualBox
* Give it OpenSSH during the install process for ease of use and fun-ness
* Log in via a terminal emulator
* Take 'er from the top and start installing/configuring stuff
* See if you get an Islandora 8 site out the other end that is functionally similar or the same as the one the Ansible playbook gets you (albeit with some missing functionality like Matomo)

# Additional Notes:
In general, it shouldn't contain lies, inaccuracies, missing steps, or unnecessary steps.

Of course, it also needs to conform to the [documentation style guide](https://islandora.github.io/documentation/technical-documentation/docs_style_guide/)
